enh: add template messages docs (#7)
callbellchat/callbell-public-docs: docs/api_reference/messages_api/post_send_messages.md

| `type`    | MessageType    | Type of message to be sent           |
| `content` | MessageContent | Content of the message               |

### Optional Parameters

| Parameter       | Type         | Description                                                       |
| :-------------- | :----------- | :---------------------------------------------------------------- |

> | `template_uuid` | TemplateUUID | Unique identifier of the template message                         |

Reviewer: I think we could just use a simple type here like `string`, WDYT?

```suggestion
| `template_uuid` | string | Unique identifier of the template message                         |
```

    }
  }'
```

## Send Template Messages

:::info
This is only available for accounts using the official **WhatsApp Business API** integration.
:::

> You can use the API to send an approved [Template](/api_reference/object_types/template) Message

Reviewer: ```suggestion
You can use the API to send an approved [Template](/api_reference/object_types/template) Message.
```

> In order to send template messages `template_uuid` and `optin_contact` **must** be present in the payload.

Reviewer: Probably this could be warning banner?

```suggestion
:::warn
In order to send template messages `template_uuid` and `optin_contact` **must** be present in the payload.
:::
```

> Use the [Templates API](/api_reference/template_messages_api/introduction) to the get the `template_uuid`s

Reviewer: Maybe this could be moved at the end of the section? In here it's slightly confusing IMO

```bash title=request.sh
curl -X POST "https://api.callbell.eu/v1/messages/send" \
  -H "Authorization: Bearer test_gshuPaZoeEG6ovbc8M79w0QyM" \
  -H "Content-Type: application/json" \
  -d '{
    "to": "+31612345678",
    "from": "whatsapp",
    "type": "text",
    "content": {
      "text": "John Doe"
    },
    "template_uuid": "d980fb66fd5043d3ace1aa06ba044342",
    "optin_contact": true
  }'
```

> In this context `text` refers to the placeholder of the template message, let's say that you have a template message like this:

Reviewer: ```suggestion
In this context `text` refers to the placeholder of the template message, for example let's say you have a template message like this:
```

```bash title=template_example
Hello {{1}}, this is a template message example
```

> After the request above the sent message will be the following:

Reviewer: ```suggestion
The placeholder replacement will be done with the value passed in the payload, so in this case it will be the following:
```

```bash title=template_example
Hello John Doe, this is a template message example
```

## Send Template Messages with Media Attachments

:::info
This is only available for accounts using the official **WhatsApp Business API** integration.
:::

You can use the API to send an approved [Template](/api_reference/object_types/template) Message

> In order to send template messages `template_uuid` and `optin_contact` must be present in the payload.

Reviewer: Same here, I think a warn banner could work better for this piece of information:

```suggestion
:::warn
In order to send template messages `template_uuid` and `optin_contact` must be present in the payload.
:::
```
